### ISSUE TITLE
Including rebol file into host-repl.r (via do, load or import) produces a ASAN/Valgrind error

### ISSUE THREAD

**Issue by draegtun** [metaeducation/ren-c#440]
This is the error produce when starting ./r3 binary:

    C Source File ../src/os/host-main.c, Line 896
    unmanaged series was likely created during evaluator tick: 24329
    series guard didn't trigger ASAN/Valgrind trap
    :
    either not a REBSER, or you're not running ASAN/Valgrind

Here is diff of my change:

    diff --git a/src/os/host-repl.r b/src/os/host-repl.r
    index 0dee170..e77bc44 100644
    --- a/src/os/host-repl.r
    +++ b/src/os/host-repl.r
    @@ -223,3 +223,5 @@ upgrade: procedure [
     ][
         fail "Automatic upgrade checking is currently not supported."
     ]
    +
    +do %/Users/barry/.rebol/repl-skin.reb

File is hardcoded at moment and does exist.  Get error regardless of content in file.

**Comment by hostilefork** [comment]
Ah, I see.

You haven't actually allowed the startup function to run yet.

You are putting your behavior in the script as a whole, at top level, where it's really supposed to just be definitions.  This means you're missing important initialization for ports.

If you have a one-off thing to do, do it in the startup function... not at global scope before the startup function runs

https://github.com/metaeducation/ren-c/blob/bf44d8a4b2e0afeb074b0dbcfb8a9c3b0165274f/src/os/host-start.r#L223

Basically, the problem is, you were running a DO on a FILE! port before the ports were registered via that function, so it's just a matter of being too early in the bootstrap.
